hpke_test.go

```diff
@@ -476,14 +490,47 @@ func verifyParameters(tv testVector, ctx cipherContext) {
 	assertBytesEqual(tv.t, tv.suite, "Incorrect parameter 'exporterSecret'", tv.exporterSecret, ctx.exporterSecret)
 }
 
+func verifyPublicKeysEqual(tv testVector, pkX, pkY KEMPublicKey) {
+	pkXm := mustSerializePub(tv.suite, pkX)
+	pkYm := mustSerializePub(tv.suite, pkY)
+	assertBytesEqual(tv.t, tv.suite, "Incorrect public key", []byte(pkXm), []byte(pkYm))
```

Comment:
Note for later: For other Go projects, I've been replacing custom assertions with github.com/stretchr/testify/require